packages/components/src/components/header/header.stories.ts

```diff
@@ -7,9 +7,21 @@ const { argTypes, parameters } = storybookDefaults('sd-header');
 const { generateTemplate } = storybookTemplate('sd-header');
 const { overrideArgs } = storybookHelpers('sd-header');
 
+/**
+ * Provides a container for the main navigation of a website.
+ *
+ *  **Related templates**
+ * - [Header Sample](?path=/docs/templates-header--docs)
+ *
+ *
+ *  **Related components**
+ * - [sd-navigation-item](?path=/docs/components-sd-navigation-item--default)
```

Comment:
```suggestion
 * - [sd-navigation-item](?path=/docs/components-sd-navigation-item--docs)
```

Should this not link to the docs page? Linking straight to the default feels a bit strange since it is just the navigation with no information or controls.